packages/core/src/checkbox/checkbox.stories.ts

```diff
@@ -0,0 +1,533 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/checkbox/register.js';
+import { html } from 'lit-html';
+import { spreadProps, getElementStorybookArgs, getElementStorybookArgTypes } from '@clr/core/internal';
+import customElements from '../../dist/core/custom-elements.json';
+
+export default {
+  title: 'Forms (Preview)/Checkbox/Stories',
+  component: 'cds-checkbox',
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=20%3A1',
```

Comment:
nit: would it be better to specify the node such that a checkbox is highlighted? 
https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=0%3A871

Not a big deal though, as the url you have will still display checkboxes.

Reply:
Updated all the inputs to highlight the correct input in figma